src/Sensors.cpp

```diff
@@ -970,6 +975,26 @@ void Sensors::GCJA5Read() {
     unitRegister(UNIT::PM10);
 }
 
+
+void Sensors::DFRobotGravityRead() {
+     String gastype = dfr_gas.queryGasType();
+  /**
+   *Fill in the parameter readGasConcentration() with the type of gas to be obtained and print
+   *The current gas concentration
+   *Print with 1s delay each time
+   */
+  Serial.print("Ambient ");
+  Serial.print(gastype);
+  Serial.print(" concentration is: ");
+  Serial.print(dfr_gas.readGasConcentrationPPM());
+  if (gastype == "O2")
+    Serial.println(" %vol");
+  else
+    Serial.println(" PPM");
+  Serial.println();
```

Comment:
We only have here some prints. We Should change these to update the co and nh3 variables @roberbike